docs/architecture.rst

```diff
@@ -0,0 +1,84 @@
+Architecture
+============
+
+The Django Debug Toolbar is designed be flexible and extensible for
```

Comment:
*to* be flexible.

Just noticed this one item. Good architecture doc. 👍

Reply:
Thank you Rob!